dev-tools/run.sh

```diff
@@ -36,7 +36,11 @@ echo -e "Starting WebPack dev server in background..." | print_info | print_pref
 deescalate_privileges npm run dev 2>&1 | print_prefix "webpack" 36 &
 
 # Waiting for initial WebPack dev build
-while [[ ! -f "${BASE_DIR}/src/cms/static/main.js" ]]; do sleep 1; done
+compgen -G "${BASE_DIR}/src/cms/static/main.*.js" > /dev/null
+while [[ $? -eq 1 ]]; do 
+    sleep 1
+    compgen -G "${BASE_DIR}/src/cms/static/main.*.js" > /dev/null
+done
```

Comment:
Is this loop intended to wait for a user fixing something? Or do we expect that the command simply fails sometimes and we need to try again?

Reply:
This loop is only entered, when webpack compilation is not finished at this moment i.e. the frontend part is ready to use
The previous version was just a bash test command, checking if webpack is ready: 
```
[[ ! -f "${BASE_DIR}/src/cms/static/main.js" ]]
```
But as far as I know the command is not appropriate for glob/regex patterns like main._some-hash_.js,
so I replaced it by this compgen, which serves the same purpose [https://stackoverflow.com/questions/6363441/check-if-a-file-exists-with-wildcard-in-shell-script](https://stackoverflow.com/questions/6363441/check-if-a-file-exists-with-wildcard-in-shell-script), i.e. compgen returns exit status 1 as long as the file does not exists

Reply:
Ah okay, waiting for `webpack`. Thanks, that helps me understand.